.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -7,9 +8,5 @@ assignees: ''
 ---
 
 **Please describe the what should be documented
-<!-- A clear and concise description of what scenario should be documented and why. -->
 
-**Suggestion for where the documentation should live. Should it be part of an existing 
-file or directory? Should there be a new file? Should it be part of the demo? Should it be
-part of the help text for the CLI?**
-<!-- A path or link here -->
+**Please suggest where in the repo should be documented
```

Comment:
Suggestion:
Please suggest where in the repo the document should be located

Reply:
thank you